### PR TITLE
Fastp known adapters formatted

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

File: qp_fastp_minimap2/qp_fastp_minimap2.py [hunk 1]
```diff
@@ -20,15 +21,15 @@
...
+FASTP_BASE = 'fastp -l 100 -i %s -w {nprocs} --adapter_fasta {adapter_fasta}'
 MINIMAP2_BASE = 'minimap2 -ax sr -t {nprocs} {database} - -a '
 SAMTOOLS_BASE = 'samtools fastq -@ {nprocs} -f '
 
 FASTP_CMD = ' '.join([FASTP_BASE, '-I %s -o {out_dir}/%s -O {out_dir}/%s'])
 FASTP_CMD_SINGLE = (f'{FASTP_BASE} -o '
                     '{out_dir}/%s')
-COMBINED_CMD = (f'{FASTP_BASE} -I %s --stdout | {MINIMAP2_BASE} | '
-                f'{SAMTOOLS_BASE} 12 -F 256 -1 '
+COMBINED_CMD = (f'{FASTP_BASE} -I %s --detect_adapter_for_pe --stdout | '
```
Comment (wasade) [inline qp_fastp_minimap2/qp_fastp_minimap2.py]: Per the out-of-band email, I do not believe we should rely on this

Comment (antgonza) [reply qp_fastp_minimap2/qp_fastp_minimap2.py]: I'm not sure what you mean, could you clarify? Note that the "--adapter_fasta {adapter_fasta}" is added to all fastp commands (line 24) and the "--detect_adapter_for_pe" is added when using pe.

Comment (wasade) [reply qp_fastp_minimap2/qp_fastp_minimap2.py]: If you prefer to use this strategy, can you explicitly verify that the known adapters are also filtered in PE mode?

Comment (antgonza) [reply qp_fastp_minimap2/qp_fastp_minimap2.py]: Discussed via email. Basically we need to remove "--detect_adapter_for_pe", which I thought it was required; thus, the disconnect. Will push the change once tests pass in my local computer. Thank you @wasade.

**Review by charles-cowart — APPROVED** [review]
Approved! Everything appears in order.

**Pull request merged**